internify/src/components/organisms/styles/Footer.css

```diff
@@ -5,7 +5,6 @@
   background-color: white;
   box-shadow: 0px 11px 20px 1px rgba(0, 0, 0, 0.45);
   overflow: hidden;
-  position: absolute;
```

Comment:
this style needs to be maintained in order to force the footer to be at the bottom for the pages that are not `Landing`